apps/ledger-live-desktop/src/renderer/components/StepperNumber.tsx

```diff
@@ -131,7 +135,7 @@ class StepperNumber extends PureComponent<Props, State> {
         <Btn onMouseDown={!isMin ? this.handleMouseDown("decrement") : undefined} disabled={isMin}>
           {"-"}
         </Btn>
-        <Num>{value}</Num>
+        <Input style={{ maxWidth: 50 }} value={value.toString()} onChange={this.onChange} />
```

Comment:
I don't have a big problem with the style prop but it's probably better to keep the uniformity with the rest of the code and use styled component for it if possible